rule/value.go

```diff
@@ -17,6 +17,7 @@ package rule
 
 import (
 	"fmt"
+	"github.com/bazelbuild/bazel-gazelle/label"
```

Comment:
I can't remember whether we have tooling for that, but please move this import down into the separate block below.